### PR TITLE
Rxtouch - remove debugging messages from UI (second revision)

### LINKED ISSUE #128: Hannaford & StopAndShop: Remove "(code 1)"
Please see the following excerpt from the out.json file:

> {\"name\":\"Hannaford (Uxbridge)\",\"hasAvailability\":false,**\"extraData\":\"No available appointments (code 1).\"**,\"availability\":{},\"signUpLink\":\"https://hannafordsched.rxtouch.com/rbssched/program/covid19/Patient/Advisory\",\"street\":\"158 No. Main St. Suite 3\",\"city\":\"Uxbridge\",\"zip\":\"01569\",\"timestamp\":\"2021-03-07T19:30:25+00:00\",\"latitude\":42.0820312,\"longitude\":-71.6390782},

The "extraData" field is displayed by the front-end to the user exactly as it is sent in the json file.  We don't need this information in "extraData" at all because the front-end will display the appropriate message since "hasAvailability" is false.

image

### PR DESCRIPTION
**Pull request by zpeyton**
Closes #128

### CODE REVIEW

**Review by tkjones117 — APPROVED** [review]

**Comment by harcod** [comment]
This fixes my issue, however, I think you are losing the "debug" information.  **IF** you want this information in the out.json file, then you have to return it in the response.  See _Hannaford/index.js, line 14_... this is where the **message** key get emitted.

**Comment by tkjones117** [comment]
Good call @harcod ! It would be a good idea to emit the debug info in Hannaford and S&S, too.

File: lib/RxTouch.js [hunk 1]
```diff
@@ -150,6 +152,7 @@ async function ScrapeRxTouch(browser, site, siteName) {
             );
             if (!facilityId) {
                 results[loc.zip] = {
+                    debug: "No available appointments. (Code 1)",
                     message: "No available appointments.",
```
Comment (harcod) [inline lib/RxTouch.js:156]: We don't need the "message" here.

**Review by harcod — CHANGES_REQUESTED** [review]
Everything looks good except for this one line!  The front-end will take care of this message without explicitly sending it in the response.  It's OK to leave "code 1" though.

**Review by harcod — APPROVED** [review]

**Pull request merged**